### ISSUE TITLE
Pre-Poly Product Polish

### ISSUE THREAD

**Issue by mr-michael** [tallycash/extension#1352]
**Related issues**
- #1359
  - #1368 
  - #1302 
  - #1169 
- #1350 
  - #1563 
  - #1564 

**Bugs** (incorporating comments below)
- [x] open sea... do we have the ticket for this
  > This has been fixed in https://github.com/tallycash/extension/pull/1491 - Greg
- [x] https://github.com/tallycash/extension/issues/1364
- [x] https://github.com/tallycash/extension/issues/1367
- [x] https://github.com/tallycash/extension/issues/1362
- [x] https://github.com/tallycash/extension/issues/1357
- [x] https://github.com/tallycash/extension/issues/1356
- [x] https://github.com/tallycash/extension/issues/1329
- [x] https://github.com/tallycash/extension/issues/1087
- [x] https://github.com/tallycash/extension/issues/1380
- [x] https://github.com/tallycash/extension/issues/1158
- [x] https://github.com/tallycash/extension/issues/1439
- [x] https://github.com/tallycash/extension/pull/1494
- [x] https://github.com/tallycash/extension/issues/1067
- [x] https://github.com/tallycash/extension/issues/998
- [x] https://github.com/tallycash/extension/issues/707
- [x] https://github.com/tallycash/extension/issues/858
- [x] https://github.com/tallycash/extension/issues/1413
  > this is almost done - Greg
- [x] https://github.com/tallycash/extension/issues/1158
- [x] https://github.com/tallycash/extension/issues/1439
- [x] https://github.com/tallycash/extension/issues/1405
- [x] https://github.com/tallycash/extension/issues/1213
- [x] https://github.com/tallycash/extension/issues/1391
- [x] https://github.com/tallycash/extension/issues/1432
- [ ] https://github.com/tallycash/extension/issues/1399
- [ ] https://github.com/tallycash/extension/issues/1160
- [x] https://github.com/tallycash/extension/issues/1251
- [x] https://github.com/tallycash/extension/issues/1595
- [x] https://github.com/tallycash/extension/issues/1596
- [x] https://github.com/tallycash/extension/issues/1590
- [x] https://github.com/tallycash/extension/issues/1593
- [x] https://github.com/tallycash/extension/issues/1595
- [x] https://github.com/tallycash/extension/issues/1597
- [x] https://github.com/tallycash/extension/issues/1598
- [x] https://github.com/tallycash/extension/issues/1599
- [x] https://github.com/tallycash/extension/issues/1601
- [x] https://github.com/tallycash/extension/issues/1602
- [x] https://github.com/tallycash/extension/issues/1587
- [x] https://github.com/tallycash/extension/issues/1665
- [x] https://github.com/tallycash/extension/issues/1664
- [x] https://github.com/tallycash/extension/issues/1653
- [ ] #1734

**Comment by 0xDaedalus** [comment]
As a starting point - here is a list of issues that I think can fall into this bucket.


[Sending "max" ETH ... doesn't](https://github.com/tallycash/extension/issues/858)
[Can't see my USDT on Tally I sent there.](https://github.com/tallycash/extension/issues/1364) 
[Swap- Gas Limit should be preset when user edits Gas Limit](https://github.com/tallycash/extension/issues/1367)
[Overview not showing all Assets](https://github.com/tallycash/extension/issues/1362)
[Broken gas prices (?!)](https://github.com/tallycash/extension/issues/1357)
[Visual jitter on ENS name lookup](https://github.com/tallycash/extension/issues/1356)
[Enforce minimum password strength](https://github.com/tallycash/extension/issues/1329)
[https://github.com/tallycash/extension/issues/1087](https://github.com/tallycash/extension/issues/1087)

**Comment by mr-michael** [comment]
Additional bugs:

https://github.com/tallycash/extension/issues/1380
https://github.com/tallycash/extension/issues/1158
https://github.com/tallycash/extension/issues/1439

**Comment by mr-michael** [comment]
https://www.flowdock.com/app/cardforcoin/tech/threads/tKvSG6JOc1R_7kmGNpECAXZPKbV

**Comment by mr-michael** [comment]
open sea, dapp integrations,

**Comment by mr-michael** [comment]
https://github.com/tallycash/extension/pull/1494

**Comment by mhluongo** [comment]
I think most of "Needs Discussion" should be ticketed and roadmapped out, but shouldn't block any releases, IMO. In particular...

> What's our plan w/randomly generated dog names? Are we happy with these? FWIW it would be far more useful for the user to see the first/last values of their address in the top nav (vs. a dog name)
> Adding a locked/unlocked indicators (and toggle... and making the timeout editable)
> Can we support a full logout option?

... are full-fledged product features that deserve a feature pitch.

> Adding custom tokens. Do we need to support this?

This might be a blocker, but I don't have a good sense of how happy users are with the current token balance checking performance. It'd be nice to have data on this from release to release... right now we get occasional complaints, but the loudest seem to have stopped. Unsure if that's because they're exhausted, or their problems have gone away :)
    
> Nano Plus support... what's the lift?

I think this is a single constant update, but I've been waiting on a test device to confirm. It could be 4 hours of fix + testing, it could be a couple weeks.

**Comment by mhluongo** [comment]
Aha! Thanks, I wonder this was fixed in https://github.com/tallycash/extension/pull/1511 ?

**Comment by mr-michael** [comment]
> > open sea... do we have the ticket for this
> 
> We've fixed 2 issues with OpenSea
> 
> #790 #1242
> 
> There might be other issues actually using the exchange? But none ticketed. Maybe @SJQuality can lean in here

Re: OpenSea @mhluongo to confirm: suggestion is for @SJQuality to poke around on [OpenSea](https://opensea.io/) (trading, bidding, etc.) to generate more tickets?

**Comment by greg-nagy** [comment]
https://github.com/tallycash/extension/issues/1665

**Comment by mhluongo** [comment]
We have Polygon now! :tada:
